tools/github/write_current_source_version.py

```diff
@@ -0,0 +1,49 @@
+# This script produces SOURCE_VERSION file with content from current version commit hash. As a
+# reminder,SOURCE_VERSION is required when building Envoy from an extracted release tarball
+# (non-git). See: bazel/get_workspace_status for more information.
+#
+# The SOURCE_VERSION file is produced by reading current version tag from VERSION.txt file then
+# fetch the corresponding commit hash from GitHub.
+#
+# Note: This script can only be executed from project root directory of an extracted "release"
+# tarball.
+
+import sys
+import json
+
```

Comment:
nit: all of these imports are stdlib so i think no need for a space

also can we use alpha ordering for imports

Reply:
Resolved.